.github/workflows/build_idf_examples.yml

```diff
@@ -11,18 +11,38 @@ jobs:
     strategy:
       matrix:
         idf_ver: ["release-v5.0", "release-v5.1", "release-v5.2", "release-v5.3", "release-v5.4", "latest"]
+        target: ["esp32s2", "esp32s3", "esp32p4"]
```

Comment:
Previously, we build all targets in one job. Now we have one job for each target, which is slower.

Could you please check if `idf-build-apps --target all` can handle the idf version correctly? Eg. it will not build for esp32p4 on idf release which do not support P4?

Please test this locally, so we do not onverload the test runners...

Reply:
Just confirmed this works. `idf-build-apps --target all` on idf v5.1.2 will not list ESP32-P4 target